### DISCUSSION TITLE
<Select.Root>

### DISCUSSION THREAD

**Discussion by Tihomir971** [huntabyte/shadcn-svelte#292] [Help]
How to catch changes in <Select.Root> component?

**Comment by Tihomir971** [comment]
> How to catch changes in <Select.Root> component?

I find it "onSelectedChange".

Documentation could be better with all Props, Events and Typedefs

**Reply by vigneshksaithal** [reply]
Hi @Tihomir971 how did you bind the value to a variable?

**Reply by Tihomir971** [reply]
<input hidden {name} type="text" value={$selected?.value} /> or use onSelectedChange event.
You can check at Melt UI. It is used by Shadcn

**Reply by vigneshksaithal** [reply]
Ok @Tihomir971 I will check Melt UI documentation. Thank you for helping.
